packages/components/package.json

```diff
@@ -69,6 +69,7 @@
     "extends @localyze-pluto/browserslist-config"
   ],
   "dependencies": {
+    "@radix-ui/react-progress": "^1.0.1",
```

Comment:
Just wondering if it should be a dependency or a peerDependency :thinking:

Reply:
Moved to peerDependency.